content/books/2026-08-15-infinite-archive-the-midsolar-murders-3/index.md

```
@@ -0,0 +1,35 @@
+title_without_series: Infinite Archive
+translationKey: 'Infinite Archive (The Midsolar Murders, #3)-B0CW1HHVD5'
+yaml: infinite-archive-the-midsolar-murders-3
+---
+
+In 'Infinite Archive,' part of the Midsolar Murders series, we step into the life of Mallory Viridian, an amateur sleuth who has become accustomed to the dramatic flair of crime on Station Eternity. After a period of relative calm without any murders—a dramatic break from her often turbulent reality—Mallory finds herself growing restless. Just as she settles into a routine, everything changes with the imminent arrival of the 'Metis,' a revolutionary data ship bringing the entire Internet from Earth and an exciting mystery fan convention to her corner of the galaxy.
+
+As anticipation builds, Mallory is unexpectedly assigned as the keynote speaker at the convention, a role she has mixed feelings about. Her literary agent arrives with the ship, and she hopes his presence will reignite her stagnant career. However, in a twist of fate, tragedy strikes when she discovers her agent dead. With a new and enigmatic alien operating in the shadows, and the specter of a powerful and unknown killer lurking at the convention, Mallory is swiftly drawn into a web of intrigue that challenges not just her instincts but her very survival.
+
+Mur Lafferty, a celebrated author known for blending science fiction with mystery, skillfully weaves together the mundane with the extraordinary in this third installment of the series. With an average reader rating of 3.86, 'Infinite Archive' resonates with those who appreciate complex narratives driven by engaging characters, offering a playful yet poignant exploration of technology and human connections.
+
+The book makes for an insightful read for fans of cozy mysteries and science fiction alike, bridging the gap between the worlds of crime-solving and futuristic lore. With 346 pages of enthralling content, Lafferty continues to prove her prowess in captivating her audience, inviting them on yet another thrilling adventure that pushes the boundaries of genre convention.
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)